app/src/main/AndroidManifest.xml

```diff
@@ -2,13 +2,42 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="io.github.polysmee">
 
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+
+    <uses-feature android:name="android.hardware.type.watch" />
+
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Polysmee">
+        <activity
+            android:name=".login.LoginCheckActivity"
+            android:label="@string/title_activity_login_check">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+
+        <!--
+               Set to true if your app is Standalone, that is, it does not require the handheld
+               app to run.
+        -->
```

Comment:
Comment in the manifest ?

Reply:
oh okay that's an auto import from studio, i'll remove it